superset/examples/birth_names.py

```diff
@@ -144,14 +145,17 @@ def load_birth_names(only_metadata=False, force=False):
                 granularity_sqla="ds",
                 compare_lag="5",
                 compare_suffix="over 5Y",
+                metric=metric,
             ),
         ),
         Slice(
             slice_name="Genders",
             viz_type="pie",
             datasource_type="table",
             datasource_id=tbl.id,
-            params=get_slice_json(defaults, viz_type="pie", groupby=["gender"]),
+            params=get_slice_json(
+                defaults, viz_type="pie", groupby=["gender"], metrics=metrics
```

Comment:
During testing I noticed that the Gender chart wasn't rendering correctly:
![image](https://user-images.githubusercontent.com/33317356/76166582-3ca28b80-6168-11ea-8c93-7d17f5876a83.png)
As the pie chart wants the singular `metric`, you need to change `metrics=metrics` to `metric=metric` here.

Reply:
good catch, thanks for deep testing this